frontend/models/Users.php

```diff
@@ -187,4 +218,54 @@ public function getAverageRating()
             return null;
         }
     }
+
+    /**
+     * Возвращает id исполнителей, у которых есть отзывы.
+     *
+     * @return array массив id исполнителей, у которых есть отзывы
+     */
+    public function getUserIdExecutor()
+    {
+        $i = 0;
+
+        $id = [];
+
+        $executorId = Reviews::find()->select('user_id_executor')->asArray()->all();
+
+        foreach($executorId as $executor) {
+            foreach($executor as $value) {
+                $id[$i] = $value;
+                $i++;
+            }
+        }
```

Comment:
Попробуй вот так:
```suggestion
        $id = Reviews::find()->select('user_id_executor')->distinct()->column();
```

Обрати внимание на distinct, т.к. в твоём варианте значения бы повторялись